USC-SIkJalpha Round 0 submission (#4)
covid19-forecast-hub-europe/covid19-scenario-hub-europe: data-processed/USC-SIkJalpha/metadata-USC-SIkJalpha.txt

team_name: University of Southern California
model_name: SI_kJalpha

> model_abbr: USC-SI_kJalpha

Reviewer: ```suggestion
model_abbr: USC-SIkJalpha
```

model_version: 2022-03-13
model_contributors: Ajitesh Srivastava <ajiteshs@usc.edu>
website_url: https://scc-usc.github.io/ReCOVER-COVID-19
license: mit
team_model_designation: primary

> methods: Uses SIKJalpha which models temporally varying infection, death, and hospitalization rates. Learning is performed by reducing the problem to multiple simple linear regression problems. Effects of variants are modeled separately as competing contagions.

Reviewer: Methods should be under 200 characters. Feel free to expand in the `methods_long` section.

Author: Hi Hugo. Sorry for the late response. I was traveling and now catching up with deadlines. I have accepted your suggestions. For this one, could you please remove the last sentence? 
Thanks,
Ajitesh

modeling_NPI: No NPI included, contact changes over time derived from Cuebiq data.
compliance_NPI: Not modeled.

> contract_tracing: Not applicable.

Reviewer: ```suggestion
contact_tracing: Not applicable.
```

testing: Not applicable.
vaccine_efficacy_transmission: Not explicitly modeled
vaccine_efficacy_delay: 14 days
vaccine_hesitancy: A "suceptible-infected"model. Boosters are modeled such that the "susceptible" population is the eligible population who are yet to get a booster. To achieve the desired saturation level to create sub-scenarios, we increase the adoption rate over time.

> vaccine_immunity_duration: Only waning immunity is considered! The probability of transferring to partially immune state by at time t is modeled as a gamma distribution such that: (i) the median is as per the scenario; (ii) the efficacy (given the partial immune protection against infection) after the first 60 days is the "initial" vaccine efficacy.

Reviewer: ```suggestion
vaccine_immunity_duration: 'Only waning immunity is considered! The probability of transferring to partially immune state by at time t is modeled as a gamma distribution such that: (i) the median is as per the scenario; (ii) the efficacy (given the partial immune protection against infection) after the first 60 days is the "initial" vaccine efficacy.'
```

natural_immunity_duration: Same as above
case_fatality_rate: Learned from recent data.
infection_fatality_rate: Learned from recent data.
asymptomatics: Similar to the range obtained for the US states using the CDC sero-prevelance data from the CDC. Note that this covers the currect cumulative fraction of unreported, untested, and asymptomatic.
age_groups: No stratification by age
importations: Not applicable
confidence_interval_method: We generate multiple trajectories by (i) Uniform sampling of the recently seen infection rates, under-reporting/asymptomatic population, vaccine coverage, waning parameters uncertainty, and variant prevelance. (ii) For each of the above case trajectories, we use multiple possibilities of death and hospitalization rates that have been seen in the recent past. The above results in multiple points for each day. Quantiles are generated based on sampling among these points for each day.

> calibration: All calibrations using regression. No manual-tuning other than the listed assumption and trivial settings: non-zero lag between infection and death.

Reviewer: ```suggestion
calibration: 'All calibrations using regression. No manual-tuning other than the listed assumption and trivial settings: non-zero lag between infection and death.'
```

spatial_structure: Not applicable.
team_funding: National Science Foundation Award # 2135784
data_inputs: Cases and deaths - JHU

> citation: 
> 	[1] Ajitesh Srivastava, Tianjian Xu and Viktor K. Prasanna, "Fast and Accurate Forecasting of COVID-19 Deaths using the SIkJalpha Model". https://arxiv.org/abs/2007.05180.

Reviewer: ```suggestion
citation: |
  [1] Ajitesh Srivastava, Tianjian Xu and Viktor K. Prasanna, "Fast and Accurate Forecasting of COVID-19 Deaths using the SIkJalpha Model". https://arxiv.org/abs/2007.05180.
```

> methods_long: [Note that this is general description. Hospitalizaitons and age-groups are not applicable to European Scenario Modeling Hub] We use our own epidemic model called SI-kJalpha, preliminary version

Reviewer: ```suggestion
methods_long: |
  [Note that this is general description. Hospitalizaitons and age-groups are not applicable to European Scenario Modeling Hub] We use our own epidemic model called SI-kJalpha, preliminary version
```
